Sources/Datadog/Datadog.swift

```diff
@@ -104,7 +105,7 @@ public class Datadog {
     /// Verbosity level of Datadog SDK. Can be used for debugging purposes.
     /// If set, internal events occuring inside SDK will be printed to debugger console if their level is equal or greater than `verbosityLevel`.
     /// Default is `nil`.
-    public static var verbosityLevel: LogLevel? = nil
+    public static var verbosityLevel: CoreLoggerLevel? = nil
```

Comment:
Note that verbosity now uses levels from core and no longer from the Log Feature.